preupg/utils.py

```diff
@@ -862,3 +848,74 @@ def run_generate(self, xml_file, html_file, old_style=False):
         return ret_val
 
 
+class ModulSetUtils(object):
+
+    @staticmethod
+    def check_property(ini_file_path, key, section):
```

Comment:
Fixed in https://github.com/upgrades-migrations/preupgrade-assistant/pull/233/commits/107a550a8ec5aeb8ddcbd5e59fef0d6a99e3e0a2